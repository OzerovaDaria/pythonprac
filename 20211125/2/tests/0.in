÷òî ÿ êàê áóäòî áû óæå ñòî òðèëëèîíîâ ìèëëèàðäîâ ëåò ïðîæèâàþ
