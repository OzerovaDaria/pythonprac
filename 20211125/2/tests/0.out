что я как будто бы уже сто триллионов миллиардов лет проживаю

